newsroom/templates/company_expiry_email.html

```diff
@@ -0,0 +1,7 @@
+<h3>The following company accounts are due to expire within next 7 days (before {{expiry_date.strftime('%Y-%m-%d')}}):</h3>
+
+<ul>
+    {% for company in companies %}
+        <h3><pre>&nbsp;{{ company.get('name') }} ({{ company.get('contact_email') }}), Tel: ({{ company.phone }}) expiring on {{ company.expiry_date.strftime('%Y-%m-%d')}}</pre></h3>
```

Comment:
Check for the existence of `contact_email` and `phone` in the template. Otherwise it will render email as `AAP (), Tel: () expiring in 25/02/2019`.

Reply:
I thought of keeping it this way so that the administrators can be alerted that the company is missing these details. Can we use (-) instead of () ? Or check for existence and not show if it doesn't exist ?

Reply:
or change the template to AAP (Email: - , Tel: -) expiring on <date>